test.html

```diff
@@ -0,0 +1,140 @@
+<p>RepoLib is a Python library and CLI tool-set for managing your software
```

Comment:
Was this meant to be committed? Presumably not given the name and the fact it appears to be generated.

Reply:
Nope, good catch.